src/views/dialog/confirm.php

```diff
@@ -25,8 +25,8 @@
 
 		<?php echo $content; ?>
 		<div class="tribe-dialog__button_wrap">
-			<button class="tribe-button tribe-confirm__cancel"><?php echo esc_html( $cancel_button_text ); ?></button>
-			<button class="tribe-button tribe-confirm__continue"><?php echo esc_html( $continue_button_text ); ?></button>
+			<button class="tribe-dialog__button tribe-dialog__button-cancel tribe-common-c-btn-border tribe-common-c-btn-border--alt"><?php echo esc_html( $cancel_button_text ); ?></button>
```

Comment:
Would it make sense to have these classes coming from the template vars and use `tribe_classes`? I can see some scenarios where we may want the button styles to be inverse of how they are right now. (i.e.: We may want the OK/Continue to have the accent color as bg color).

Reply:
Excellent idea. Addressed!